execution/executor-types/src/lib.rs

```diff
@@ -153,7 +153,9 @@ impl StateComputeResult {
 
 impl StateComputeResult {
     pub fn version(&self) -> Version {
-        max(self.num_leaves, 1) - 1
+        max(self.num_leaves, 1)
+            .checked_sub(1)
+            .expect("Integer overflow occurred")
```

Comment:
@msmouse  :D?

Reply:
ping @lightmark as well, can we get a stamp here :)?

Reply:
sorry I missed the mention.